tasking/pulp/tasking/celery_app.py

```diff
@@ -69,6 +66,10 @@ def initialize_worker(sender, instance, **kwargs):
     :param kwargs:   Other params (unused)
     :type  kwargs:   dict
     """
+    from pulp.tasking.services.worker_watcher import delete_worker
+    from pulp.tasking.services import storage
+    from pulp.tasking.constants import TASKING_CONSTANTS
```

Comment:
It is not necessary to import `constants` and `storage` inside callables. There should be no harm if they will be in their usual place because we import `pulp.tasking` anyway: `from pulp.tasking.celery_instance import celery`